index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ]  You should have your CSS file connected to your htmll file.  I can see that you have both HTML and CSS file, but they are not connected successful.  You can read the [documentation ](https://www.w3schools.com/css/css_howto.asp)here .  and remember to use external CSS.  this will help you to have you project connected with your styling ....